tempesta_fw/ss_skb.c

```diff
@@ -700,8 +701,7 @@ __skb_fragment(struct sk_buff *skb, struct sk_buff *pskb,
 
 	if ((offset >= 0) && (offset < d_size)) {
 		unsigned int t_size = d_size - offset;
```

Comment:
Shouldn't we use here `int` type and not `unsigned int` one? Even the result is supposed to be positive, conversion from `t_size` to `-t_size` below looks suspicious.